### PR TITLE
readme updated

### PR DESCRIPTION
**Pull request by ajabble**
#3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,25 +24,20 @@ http://localhost:8118
 
 ## Request:
 
-`curl -X POST -H "Content-Type: application/json" http://localhost:8118/ingress/messages -d <payload>`
```
Comment (praveen-garg) [inline README.md]: move npm run test to another heading; and say run in diff session as these are integration test not unit

File: README.md [hunk 2]
```diff
@@ -24,25 +24,20 @@ http://localhost:8118
...
-
-see test.js
+### Valid/Invalid Block Payloads
 
+* see test.js
 
-Not sure about this one?
-
-`'{"timestamp": "2017-01-02T01:02:03.23232Z-05:00", "asset": "pump1", "key": "80a43623-ebe5-40d6-8d80-3f892da9b3b4", "readings": ["x"]}'`
+* see payload.json
```
Comment (praveen-garg) [inline README.md]: see (valid) payload.json

File: README.md [hunk 3]
```diff
@@ -52,26 +47,14 @@ Not sure about this one?
 
 `curl -X POST -H "Content-Type: application/json" http://localhost:8118/ingress/message -d <payload>`
 
-### Valid Payloads
```
Comment (praveen-garg) [inline README.md]: This looks like a duplicate; 

We may remove `http://localhost:8118/ingress/message` so just remove this section from readme

File: README.md [hunk 4]
```diff
@@ -24,25 +24,20 @@ http://localhost:8118
...
+* `curl -X POST -H "Content-Type: application/json" http://localhost:8118/ingress/messages -d @payload.json`
 
-see test.js
-
-### Invalid Block Payloads
-
-see test.js
+### Valid/Invalid Block Payloads
 
+* see test.js
```
Comment (praveen-garg) [inline README.md]: link  (relative path ) file in github

File: README.md (hunk 2 above)
Comment (praveen-garg) [reply README.md]: link  (relative path ) file in github

File: README.md (hunk 1 above)
Comment (ajabble) [reply README.md]: done

**Pull request merged**